data/pattern-lab-categories.yml

```diff
@@ -0,0 +1,34 @@
+categories:
```

Comment:
This structure isn't "set in stone", it can be different per-project. Are we accounting for that? In other words, is there a way to configure this structure per-project (i.e. in a separate, project-specific config).